src/compile/axis.ts

```diff
@@ -61,6 +61,19 @@ export function parseInnerAxis(channel: Channel, model: Model): VgAxis {
     }
   });
 
+  const props = model.axis(channel).properties || {};
+
+  // it might have more to be included here.
+  ['grid'].forEach(function(group) {
+    const value = properties[group] ?
```

Comment:
I tried and grid is only the special rule here. Should I remove the list and only leave grid here or keep it as a list?